tilecloud_chain/generate.py

```diff
@@ -501,7 +498,7 @@ def main():
     if options.detach:
         detach()  # pragma: no cover
 
-    gene = TileGeneration(options.config, options)
+    gene = TileGeneration(options.config, options, multi_thread=options.get_hash is None)
```

Comment:
is this necessary? you already pass `options` to the constructor, can't you have this check directly in `__init__` of `TileGeneration` ?

Reply:
(there are other places where this remark is also valid)

Reply:
TileGeneration is not supposed to know the option added by generate.py, it'y why I do like that :-)

Reply:
ok